android/app/src/androidTest/kotlin/net/mullvad/mullvadvpn/compose/dialog/CustomPortDialogTest.kt

```diff
@@ -0,0 +1,64 @@
+package net.mullvad.mullvadvpn.compose.dialog
+
+import android.annotation.SuppressLint
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performTextInput
+import io.mockk.MockKAnnotations
+import net.mullvad.mullvadvpn.compose.setContentWithTheme
+import net.mullvad.mullvadvpn.compose.test.CUSTOM_PORT_DIALOG_INPUT_TEST_TAG
+import net.mullvad.mullvadvpn.model.PortRange
+import net.mullvad.mullvadvpn.onNodeWithTagAndText
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class CustomPortDialogTest {
+    @get:Rule val composeTestRule = createComposeRule()
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this)
+    }
+
+    @SuppressLint("ComposableNaming")
+    @Composable
+    private fun testWireguardCustomPortDialog(
```

Comment:
Generally I think this is nicer than providing, bad/incorrect defaults to the function since it might be more error prone. E.g if we miss one argument for `onSave` we won't be getting any compile error just fail silent behavior. It is more convenient with default but also more risky. I think in this specific case we could have a default for initialPort but `onSave` could be risky.

Maybe we should align this internally in the team.

Reply:
Yes lets talk bout it.

Reply:
I have come around to your position, so I will resolve this.